src/Data/NodeResolver.php

```diff
@@ -207,7 +207,7 @@ public function resolve_uri( string $uri, $extra_query_vars = '' ) {
 
 						if ( $wp_rewrite->use_verbose_page_rules && preg_match( '/pagename=\$matches\[([0-9]+)\]/', $query, $varmatch ) ) {
 							// This is a verbose page match, let's check to be sure about it.
-							$page = get_page_by_path( $matches[ $varmatch[1] ] );
+							$page = get_page_by_path( $matches[ $varmatch[1] ] ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.get_page_by_path_get_page_by_path
```

Comment:
For this one, should we do a if/else strategy with `wpcom_vip_get_page_by_path`?